_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: akn7bn
-name: Iframe with negative tabindex has no interactive elements
+name: Iframe with interactive elements are in tab-order
```

Comment:
```suggestion
name: Interactive elements within iframe are in tab-order
```

@Jym77, per HTML specs and your previous feedback:

> iframe are not necessarily [focusable areas](https://html.spec.whatwg.org/multipage/interaction.html#focusable-area) (this is actually UA dependent due to the the "or the element is determined by the user agent to be focusable" bit 😞), and therefore are (possibly) dropped out of the flattened scope (and the final tab-order) of the full page.

from title seems that we leave browsers with the responsibility to decide if an iframe is passing or failing this rule.
In fact, if you create an iframe with a simple input as content (e.g. `<input type="text" name="test" aria-label="test">`),opening it in FireFox the iframe is tabbable (and consistent with this rule title), if instead you open it in Chrome the iframe is not tabbable (and seems not consistent with this rule title).

The final goal is to allow users interacting with the iframe content, so I would suggest to use "interactive elements" as subject of this rule.

Reply:
On the other hand, this leads to a bit of discrepancy between rule title and "test target"/applicability 🤔